libs/server/shared/core/src/lib/config/.env-sample

```diff
@@ -1,2 +1,6 @@
+// bitbucket
+PMP_API_SERVICE_BITBUCKET_TOKEN=
+
+// github
 PMP_API_SERVICE_GITHUB_TOKEN=
-PMP_API_SERVICE_REPOSITORIES=valueadd-poland/pimp-my-pr,angular/angular
+PMP_API_SERVICE_REPOSITORIES=
```

Comment:
maybe consider leaving it as it was. I wanted an example of what is the format of this variable.